sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredential.java

```diff
@@ -23,6 +27,18 @@
  */
 @Immutable
 public final class DefaultAzureCredential extends ChainedTokenCredential {
+    private static final String DEFAULT_CACHE_FILE_NAME = "msal.cache";
+    private static final Path DEFAULT_CACHE_DIRECTORY = Platform.isWindows() ?
+            Paths.get(System.getProperty("user.home"), "AppData", "Local", ".IdentityService", "msal.cache") :
+            Paths.get(System.getProperty("user.home"),".IdentityService", "msal.cache");
+    private static final String DEFAULT_KEYCHAIN_SERVICE = "Microsoft.Developer.IdentityService";
+    private static final String DEFAULT_KEYCHAIN_ACCOUNT = "MSALCache";
+    private static final String DEFAULT_KEYRING_NAME = "default";
+    private static final KeyRingItemSchema DEFAULT_KEYRING_SCHEMA = KeyRingItemSchema.GENERIC_SECRET;
+    private static final String DEFAULT_KEYRING_ITEM_NAME = DEFAULT_KEYCHAIN_ACCOUNT;
+    private static final String DEFAULT_KEYRING_ATTR_NAME = "MsalClientID";
```

Comment:
Line 39 and 40 are also from the test https://github.com/AzureAD/microsoft-authentication-extensions-for-dotnet/blob/85c1e8ec283b940b1cc34eae44ac37fa645738ff/tests/Microsoft.Identity.Client.Extensions.Msal.UnitTests/MsalCacheStorageTests.cs